packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -563,18 +568,34 @@ export default class XdslAccessCtrl {
       .$promise.then((profiles) => {
         let profilesTemplate = '';
         profiles.forEach((profile) => {
-          profilesTemplate = `${profilesTemplate}<p data-translate="xdsl_access_profile_tooltip_${profile.name.toLowerCase()}"></p>`;
+          if (!profilesTemplate && profile.name.includes('VDSL')) {
+            profilesTemplate =
+              '<p data-translate="xdsl_access_profile_tooltip_description"></p>';
```

Comment:
Would it make sense to create components instead for each profiles and just load depending on the profile ? I think it is quite difficult to maintain HTML code with that way of doing

Reply:
@marie-j I think it would make sense. It could be done into a separate Pull Request since this one received the `quality check: ok` label.

Thank you!